src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -60,13 +62,23 @@
     (catch Exception e
       (log/error (format "Error parsing %s; skipping" file)))))
 
+(defn error-if
+  [pred msg x]
+  (if (pred x) (log/error msg) x))
+
 (defn load-sample-data
   "Load all .json files contained in `dir`."
-  [dir]
-  (->> (fs/glob (fs/file dir "*.json"))
-       (map try-load-file)
-       (remove nil?)
-       (vec)))
+  ([dir]
+   (load-sample-data dir false))
+  ([dir from-classpath?]
+   (let [target-files (if from-classpath?
+                        (rest (file-seq (io/file (io/resource dir))))
+                        (fs/glob (fs/file dir "*.json")))]
+     (->> target-files
+          (map try-load-file)
+          (remove nil?)
+          (vec)
+          (error-if empty? (format "Supplied directory %s contains no usable data!" dir))))))
```

Comment:
This is writing the error to the log but below it's being written to the console via println etc. Was that intentional?